app/src/main/java/org/sopt/havit/ui/category/CategoryContentModifyActivity.kt

```diff
@@ -52,6 +54,7 @@ class CategoryContentModifyActivity :
         position = intent.getIntExtra("position", 0)
         id = intent.getIntExtra("categoryId", 0)
         categoryTitleList = intent.getStringArrayListExtra("categoryNameList") as ArrayList<String>
+        preActivity = intent.getStringExtra("preActivity").toString()
```

Comment:
와 변수명 넘 괜찮네여,, before로 해놨는데 저도 이걸로 바꿔야겠어여😆

Reply:
Key 값에 대해서 조금 첨언을 하자면, Extra 관련 메소드를 활용할 때 현재처럼 string값을 직접사용하게 되면
추후에 코드 수정이 이뤄져야 할 때 Extra 데이터를 put 해주는쪽과, set 해주는 코드 양측에서 수정이 일어나야 합니다.

뭐 제 개인 취향이긴 합니다만 저같으면 아래와 같이 사용할것 같습니다~
```suggestion
        preActivity = intent.getStringExtra(어쩌구상수이름).toString()

    companion object {
        const val 어쩌구상수이름 = "preActivity"
    }
     
```